sql/src/main/java/io/crate/auth/user/User.java

```diff
@@ -26,13 +26,18 @@
 import com.google.common.collect.ImmutableSet;
 import io.crate.analyze.user.Privilege;
 import io.crate.user.SecureHash;
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.common.io.stream.StreamOutput;
+import org.elasticsearch.common.io.stream.Writeable;
 
 import javax.annotation.Nullable;
+import java.io.IOException;
 import java.util.EnumSet;
+import java.util.HashSet;
 import java.util.Objects;
 import java.util.Set;
 
-public class User {
+public class User implements Writeable {
```

Comment:
i think its a bit overkill to stream this object completely incl roles and privileges if the only thing needed is the username.